Adding Logo To Readme

D3FaltXD opened the pull request:
Made a logo with appropriate colors for the dark mode

D3FaltXD committed:
Adding Logo To Readme

Made a logo with appropriate colors for the dark mode

rustyxlol reviewed (changes requested):
Very neat. I feel like only the logo will suffice, text seems to ruin the logo quite a bit.

D3FaltXD committed:
Removed the Name and Cleaned the logo

Made the edges finer, Cleaned the logo
and updated it in the readme. Also removed the name with it

rustyxlol reviewed (approved):
perfect
